lib/matplotlib/axes/_base.py

```diff
@@ -2405,12 +2405,12 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True):
 
         if self.use_sticky_edges and (self._xmargin or self._ymargin):
             stickies = [artist.sticky_edges for artist in self.get_children()]
-            x_stickies = sum([sticky.x for sticky in stickies], [])
-            y_stickies = sum([sticky.y for sticky in stickies], [])
+            x_stickies = np.array([x for sticky in stickies for x in sticky.x])
```

Comment:
np.isclose is called just below anyways so converting to array here vs just below shouldn't affect performance.